src/index.js

```diff
@@ -1,46 +1,16 @@
 'use strict';
 
 var _ = require('underscore'),
-    DEFAULT_RESOURCE_CONFIG;
+    resourceUtils = require('./util/resource'),
+    constants = require('./constants');
 
-DEFAULT_RESOURCE_CONFIG = {
-   AbsoluteMinimumProvisioned: 1,
-   AbsoluteMaximumProvisioned: 10,
-   MinutesOfStatsToRetrieve: 20,
-   MinutesOfStatsToIgnore: 2,
-   MinutesToForecast: 5,
-   PreferredForecastPadding: [
-      { IfGreaterThan: 0, IfLessThanOrEqual: 30, Percentage: 100 },
-      { IfGreaterThan: 30, IfLessThanOrEqual: 100, Percentage: 50 },
-      { IfGreaterThan: 100, IfLessThanOrEqual: 1000, Percentage: 25 },
-      { IfGreaterThan: 1000, IfLessThanOrEqual: 1000000, Percentage: 15 },
-   ],
-   MinimumMinutesBetweenIncreases: 0,
-   MaximumIncreaseAmount: [
-      { IfGreaterThan: 0, IfLessThanOrEqual: 10, Percentage: 100 },
-      { IfGreaterThan: 10, IfLessThanOrEqual: 100, Percentage: 50 },
-      { IfGreaterThan: 100, IfLessThanOrEqual: 1000, Percentage: 20 },
-   ],
-   MinimumMinutesBetweenDecreases: 45,
-   MinimumDecreaseAmount: [
-      { IfGreaterThan: 0, IfLessThanOrEqual: 100, Percentage: 50 },
-      { IfGreaterThan: 100, IfLessThanOrEqual: 1000000, Percentage: 30 },
-   ],
-   MinimumMinutesBeforeDecreaseAfterIncrease: 15,
-   MaximumDecreasesToUsePerDay: 4,
-};
+module.exports = {};
 
+// Expose for easy access and legacy compatibility
+_.extend(module.exports, _.pick(constants, 'READ', 'WRITE', 'DEFAULT_RESOURCE_CONFIG'));
 
-module.exports = {
-   READ: 'ReadCapacityUnits',
-   WRITE: 'WriteCapacityUnits',
-
-   DEFAULT_RESOURCE_CONFIG: DEFAULT_RESOURCE_CONFIG,
-
-   makeResourceName: function makeResourceName(tableName, indexName) {
-      return tableName + (indexName ? ('::' + indexName) : '');
-   },
-};
+// DEPRECATED: Please use this function from its actual location: `./util/resource.js#makeResourceName`
+_.extend(module.exports, _.pick(resourceUtils, 'makeResourceName'));
```

Comment:
More reasoning: let's not do this just for backwards-compatibility - let's make an API breaking change and bump the version accordingly. And, I don't think anyone outside the repo needs it, so we probably don't need to expose it at all.